sky_provider/example_dags/sky_gcp_credentials.py

```diff
@@ -0,0 +1,29 @@
+from airflow.decorators import dag
```

Comment:
style nit: we have a convention to always `import` modules instead of functions/classes.

https://google.github.io/styleguide/pyguide.html#22-imports